### DISCUSSION TITLE
3 problems with url constructions for images

### DISCUSSION THREAD

**Discussion by francoiscrouzet** [archaiodata/thacer#72] [General]
Au final @AlZorba je fait une discussion globale car on a 3 issues sur ce même sujet

Les 3 problemes sont https://github.com/archaiodata/thacer/discussions/71, https://github.com/archaiodata/thacer/issues/70 et https://github.com/archaiodata/thacer/discussions/69
Peut-être que https://github.com/archaiodata/thacer/discussions/33 est aussi lié 

ça serait l'occasion de m'expliquer comment marche le system des 3 ids

**Comment by francoiscrouzet** [comment]
Je vois 3 endroits où ces url sont creés :    

`createCeramUrl`

```
document.getElementById('nonloc').innerHTML += [
                  '<a class="unlocalised-tag px-1 m-0 border border-white" href="ceram?ID=' +
                    obj.properties.ID +
                    '&ANA=8888880&INV=88880">' +
                    label +
                    '</a>'
                ]
```

    `fetch(`${import.meta.env.VITE_API_URL}ceram.php?INV=${INV}&ANA=${ANA}`)`

**Comment by AlZorba** [comment]
@francoiscrouzet c'était ça ! ça n'est plus
